src/components/TabBar/TabBarStyles.tsx

```diff
@@ -5,18 +5,62 @@ export const styles = (theme: Theme) =>
     navbar: {
       position: 'fixed',
       width: '100%',
-      bottom: 0
+      height: '8vh',
+      left: '0%',
+      bottom: 0,
+      zIndex: 0,
     },
-    text: {
-      font: 'Roboto',
-      color: theme.palette.brown.main,
+    label: {
+      position: 'relative',
+      paddingTop: '7px',
+      fontFamily: theme.typography.fontFamily,
       fontSize: '12px',
-      fontWeight: 700,
-      '&$selected': {
-        color: theme.palette.primary.main
+      fontWeight: 'bold',
+      lineHeight: '15px',
+      textAlign: 'center'
+    },
+    suitcase: {
+      background: theme.palette.white.main,
+      height: '90px',
+      width: '90px',
+      border: '4px solid #F6B350',
+      position: 'fixed',
+      bottom: '2.5vh',
+      zIndex: 1,
+      boxShadow: '1px 2.5px 3px 1px rgba(218, 207, 207, 0.5)',
+    },
+    goldsuitcase: {
+      position: 'relative',
+      top: '12px',
+      height: '37px',
+      width: '37px',
+      alignContent: 'center',
+    },
+    buttontext: {
+      textTransform: 'capitalize',
+      position: 'relative',
+      top: '-6px',
+      fontFamily: theme.typography.fontFamily,
+      fontStyle: 'normal',
+      fontWeight: 'bold',
+      fontSize: '12px',
+      textAlign: 'center',
+      color: '#F6B350',
```

Comment:
Oo instead of hardcoding the colors, could we use the theme's predefined pallete colors? :0